addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -162,103 +163,20 @@ export class SuggestionService {
      */
     sortPartnerSuggestions(partners, searchTerm = "", thread = undefined) {
         const cleanedSearchTerm = cleanTerm(searchTerm);
-        /**
-         * Ordering:
-         * - recent chat partners
-         * - internal users
-         * - channel members
-         * - thread followers
-         * - longgest match of name, alphabetically if having same length of match
-         * - longgest match of email, alphabetically if having same length of match
-         * - id
-         */
+        const compareFnList = partnerSuggestionCompareRegistry
+            .getAll()
+            .sort(({ sequence: s1 }, { sequence: s2 }) => s1 - s2)
+            .map(({ fn }) => fn);
```

Comment:
Is the map really necessary? Could just `{ fn }` as part of the loop below?

Reply:
Or using registry sequence and having directly fn in the registry?

Reply:
That's how it is done now :)